Framework/SINQ/src/PoldiPeakSearch.cpp

```diff
@@ -583,7 +584,7 @@ void PoldiPeakSearch::exec() {
 
   g_log.information() << "   Parameters set.\n";
 
-  MantidVec summedNeighborCounts = getNeighborSums(correlatedCounts);
+  MantidVec summedNeighborCounts = getNeighborSums(correlatedCounts.rawData());
   g_log.information() << "   Neighboring counts summed, contains "
                       << summedNeighborCounts.size() << " data points.\n";
 
```

Comment:
I can't leave comments on unchanged code in a review, but the call to `findPeaks` in the line _below_ should have `.cbegin()`, `.cend()`, the function `findPeaks` takes `MantidVec::const_iterator`